libvg/src/utf8.rs

```diff
@@ -0,0 +1,29 @@
+use byond::call::return_to_byond;
+use encoding::all::WINDOWS_1252;
+use encoding::label::encoding_from_windows_code_page;
+use encoding::types::DecoderTrap;
+use libc;
+use std::ffi::CStr;
+use std::slice;
+use std::ptr::null;
+
+#[no_mangle]
+pub extern "C" fn to_utf8(n: libc::c_int, v: *const *const libc::c_char) -> *const libc::c_char {
+    if n != 2 {
```

Comment:
Yep. DLL invocation passes an amount of strings and a pointer to an array of pointers to c strings. That's why `v` is a `*const *const lib:c_char`